GAE/src/org/waterforpeople/mapping/app/web/rest/QuestionRestService.java

```diff
@@ -204,8 +203,8 @@ private static void setQuestionOptionsIdList(List<QuestionDto> questionDtoList,
         Question q = questionDao.getByKey(id);
         QuestionDto dto = null;
         if (q != null) {
-            dto = new QuestionDto();
-            DtoMarshaller.copyToDto(q, dto);
+            QuestionDtoMapper qdm = new QuestionDtoMapper();
+            dto = qdm.transform(q);
             if (q.getType() == Question.Type.OPTION) {
                 Map<Integer, QuestionOption> qoMap = questionOptionDao.listOptionByQuestion(dto
```

Comment:
I think here we should also consider using the `questionOptionDao.listByQuestionId()` so that we do not even bother with querying for translations.  We do not need for this case